### PR TITLE
Update CustomHttpClient.php

### PR DESCRIPTION
**Pull request by CorbaDigitpaint**
My intelephense linter is throwing errors that `$this->$_guzz` is undefined (using PHP v7.3). This should fix it.

### CODE REVIEW

File: examples/CustomHttpClient.php [hunk 1]
```diff
@@ -23,11 +23,11 @@ class HttpCliImp implements HttpClientInterface {
 
     public function __construct(Guzzle $guzz)
     {
-        $this->$_guzz = $guzz;
+        $this->_guzz = $guzz;
```
Comment (stanleynguyen) [inline examples/CustomHttpClient.php:26]: The attribute declaration on line 22 needs to be updated to `private _guzz` as well

**Review by stanleynguyen — CHANGES_REQUESTED** [review]
Hey @CorbaDigitpaint, thank you so much for the contribution. Could you also help to run `composer update` in your local or allow reviewer to update code directly so I can help with that?

File: examples/CustomHttpClient.php (hunk 1 above)
Comment (CorbaDigitpaint) [reply examples/CustomHttpClient.php:26]: I believe on line 22 it should still be `private $_guzz` because it is still a variable.. no?

**Comment by CorbaDigitpaint** [comment]
I just ran `composer update` but I am not sure why the checks fail.. unfortunately, I don't have much time to like into this further.. I have sent you an access request to this repo @stanleynguyen.

File: examples/CustomHttpClient.php (hunk 1 above)
Comment (stanleynguyen) [reply examples/CustomHttpClient.php:26]: Oh right right

**Comment by stanleynguyen** [comment]
> I just ran `composer update` but I am not sure why the checks fail.. unfortunately, I don't have much time to like into this further.. I have sent you an access request to this repo @stanleynguyen.

One of the dependency is problematic. It's been taken care of by #115

**Comment by stanleynguyen** [comment]
Thank you so much for the contrib @CorbaDigitpaint

**Pull request merged**